cli/tests/device-appsUpdateTestAll.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+GIT_REMOTE=git@github.com:LedgerHQ/ledger-live-common.git
+
+git clean -xdf &&
```

Comment:
if you add `set -e` at beginning of bash, it will have the same effect as all the `&&` so you can remove them